### PR TITLE
Add Navbar Component

### PR DESCRIPTION
**Pull request by vaiso**
* Created a Navbar component using Material UI's AppBar
* Possible TODOs for future:
  1. Refactor Navbar to be generic so that it can be used for other cases (e.g. navigation within a Project?
  2. Make Navbar collapse into a hamburger or dropdown menu for smaller displays.

### CODE REVIEW

File: src/components/Navbar/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	color: black;
+	padding: 0 2vw;
+}
+
+.nav-link.active {
+	color: darkblue;
+}
+
+.nav-title {
+	font-size: 20px !important;
```
Comment (KevinetPeng) [inline src/components/Navbar/Navbar.css:24]: Not a fan of using the !important rule. In most cases it's better to write higher specificity.

Comment (vaiso) [reply src/components/Navbar/Navbar.css]: It seems like I'm able to change the font size without using `!important` if I inline the style, but not sure if that's a good solution either. Alternatively I could add a theme provider as per [the documentation](https://material-ui.com/customization/typography/) but that seems a bit overkill just to change the font size. Do you have any thoughts on either of these?

Comment (KevinetPeng) [reply src/components/Navbar/Navbar.css]: mbmb i forgot to reply to this. 

You don't need to use inline style={...} or !important to override higher specificity in CSS (unless the MaterialUI component already applied an !important rule to the element, which should almost never occur). You should take a look at the DOM tree and see what classes are being applied to the element. 

Some examples of how you could create higher specificity selectors could be:

.<parent element classname> .nav-title { ... }

.<another class name applied to same element>.nav-title { ... }

There's a lot of different ways to do this, so it depends on what you think works best in this case. If you prefer using SASS, we can also take a look into integrating it into the project.

Comment (vaiso) [reply src/components/Navbar/Navbar.css]: Changed styling to use higher specificity as per our discussion

**Pull request merged**